API_compatible_README.md

```diff
@@ -0,0 +1,24 @@
+# API Compatible
+
+*Instructions are for setting up API for Audacious client on Ubuntu. The steps can be modified for other clients as well.*
+
+**SOURCE_1**: http://redmine.audacious-media-player.org/boards/1/topics/788
+
+1. Install dependencies from **SOURCE_1** then clone the repo and install audacious.
+2. Before installing audacious-plugins, edit the file `audacious-plugins/src/scrobbler2/scrobbler.h` to update the following settings on lines L26-L28.  
+  - `SCROBBLER_API_KEY` to the key given on "listenbrainz.org/user/import" page.
+  - `SCROBBLER_URL` to "http://last.fm/2.0/".  
```

Comment:
- Audacious has network issues while connecting to last.fm so i had to change the URL. (Was not sure if its a bug). And regarding the scrobbling URL, i realised this when testing it on a windows client. This problem can be solved by adding 2 entries in **/etc/hosts**:  
`127.0.0.1 ws.audioscrobbler`  
`127.0.0.1 last.fm`   
I'll surely add more instructions on how to setup this week.
- I was quite confused with API key. I had web authentication in mind, which requires the use of API key allocated to the user. But user should never use API_compat for scrobbling using the web-auth. LB has its own native API for that. Would be updating the same in the next commits.